ios/FluentUI/Core/Locale+Extensions.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import SwiftUI
```

Comment:
Looks good! Could you just move this to the `Extensions/` directory? 

https://github.com/sophialee0416/fluentui-apple/tree/sople/locale/ios/FluentUI/Extensions